.cirrus.yml

```diff
@@ -123,6 +126,91 @@ task:
       pg-ci-${CIRRUS_TASK_NAME}-${DATE}
 
 
+task:
+  matrix:
+    - name: openbsd-7-0-vanilla
+      env:
+        PACKERFILE: packer/bsd/openbsd_vanilla.pkr.hcl
+
+    - name: netbsd-9-2-vanilla
+      env:
+        PACKERFILE: packer/bsd/netbsd_vanilla.pkr.hcl
+
+  container:
+    dockerfile: docker/linux_debian_packer
+    cpu: 2
+    memory: 4G
+
+  skip: $CIRRUS_LAST_GREEN_CHANGE != '' && $CIRRUS_CRON != 'regular-rebuild' && !changesInclude('.cirrus.yml', docker/linux_debian_packer, files/bsd/*, 'scripts/bsd/*', $PACKERFILE)
+  auto_cancellation: false
+
+  <<: *gcp_auth_unix
+
+  info_script:
+    - packer --version
+
+  build_image_script: |
+    DATE=$(date --utc +'%Y-%m-%dt%H-%M')
+    PACKER_LOG=1
+
+    packer build \
+      -timestamp-ui \
+      -force \
+      -var "image_date=$DATE" \
+      -var "image_name=${CIRRUS_TASK_NAME}" \
+      -var "bucket=$BUCKET" \
+      -var gcp_project=$GCP_PROJECT \
+      "$PACKERFILE"
+    gcloud compute images add-iam-policy-binding \
+      --project "$GCP_PROJECT" \
+      --member=allAuthenticatedUsers \
+      --role=roles/compute.imageUser \
+      pg-ci-${CIRRUS_TASK_NAME}-${DATE}
+
+
+task:
+  matrix:
+    - name: openbsd-7-0-postgres
+      depends_on:
```

Comment:
I think these could just be part of the -vanilla tasks. The depends_on: should still work, if they're generated as part of one matrix: